<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="nova-PC" name="com.webirrse.code.common.poi.CompareExcel2" tests="1" failures="0" timestamp="19 Jun 2017 02:40:33 GMT" time="0.733" errors="1">
  <testcase name="main" time="0.733" classname="com.webirrse.code.common.poi.CompareExcel2">
    <error type="java.lang.Throwable">
      <![CDATA[java.lang.Throwable
	at @: 第3行。第2列。G:\temp\桌面临时文件\20170616\excel2.xlsx比较两个数是否相等： expected [b32] but found [a32] in method : .main(CompareExcel2:40)
	at com.webirrse.code.common.poi.CompareExcel2.compareTwoRows(CompareExcel2.java:78)
	at com.webirrse.code.common.poi.CompareExcel2.compareTwoSheets(CompareExcel2.java:60)
	at com.webirrse.code.common.poi.CompareExcel2.main(CompareExcel2.java:40)
	at @: 第4行。第1列。G:\temp\桌面临时文件\20170616\excel2.xlsx比较两个数是否相等： expected [b41] but found [a41] in method : .main(CompareExcel2:40)
	at com.webirrse.code.common.poi.CompareExcel2.compareTwoRows(CompareExcel2.java:78)
	at com.webirrse.code.common.poi.CompareExcel2.compareTwoSheets(CompareExcel2.java:60)
	at com.webirrse.code.common.poi.CompareExcel2.main(CompareExcel2.java:40)
	at @: 第4行。第3列。G:\temp\桌面临时文件\20170616\excel2.xlsx比较两个数是否相等： expected [b43] but found [a43] in method : .main(CompareExcel2:40)
	at com.webirrse.code.common.poi.CompareExcel2.compareTwoRows(CompareExcel2.java:78)
	at com.webirrse.code.common.poi.CompareExcel2.compareTwoSheets(CompareExcel2.java:60)
	at com.webirrse.code.common.poi.CompareExcel2.main(CompareExcel2.java:40)
	at @: 第4行。第4列。G:\temp\桌面临时文件\20170616\excel2.xlsx比较两个数是否相等： expected [b44] but found [a44] in method : .main(CompareExcel2:40)
	at com.webirrse.code.common.poi.CompareExcel2.compareTwoRows(CompareExcel2.java:78)
	at com.webirrse.code.common.poi.CompareExcel2.compareTwoSheets(CompareExcel2.java:60)
	at com.webirrse.code.common.poi.CompareExcel2.main(CompareExcel2.java:40)
]]>
    </error>
  </testcase> <!-- main -->
</testsuite> <!-- com.webirrse.code.common.poi.CompareExcel2 -->
